.github/workflows/firebase-hosting-prd-merge.yml

```diff
@@ -1,4 +1,7 @@
-name: Deploy to Dev on Merge With Main
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
```

Comment:
why does this still say main if its for PRD?

Reply:
Good catch